source/_integrations/meteoclimatic.markdown

```diff
@@ -0,0 +1,81 @@
+---
+title: Meteoclimatic
+description: Instructions on how to integrate Meteoclimatic within Home Assistant.
+ha_release: 0.112
+ha_iot_class: Cloud Polling
+ha_category:
+  - Sensor
+  - Weather
+ha_codeowners:
+  - '@adrianmo'
+ha_config_flow: true
+ha_domain: meteoclimatic
+---
+
+The `meteoclimatic` integration uses the [Meteoclimatic](https://www.meteoclimatic.net/) web service as a source for meteorological data for your location. The location is based on the `station_code` configured in your `configuration.yaml` file.
+
+There is currently support for the following platforms within Home Assistant:
+
+- [Sensor](#sensor)
+- Weather
+
+It displays the current weather reported by specific Meteoclimatic stations.
+
+## Setup the integration
+
+There are two ways to integrate Meteoclimatic in Home Assistant.
+
+### Via the frontend
+
+Menu: *Configuration* -> *Integrations*
+
+Search for "Meteoclimatic", add your station code, click submit, you are done!
+
+### Via the configuration file
+
+To add Meteoclimatic to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+meteoclimatic:
+  - station_code: ESCAT4300000043206B
+```
+
+{% configuration %}
+  station_code:
+    description: Station code belonging to a Meteoclimatic station.
+    required: true
+    type: string
+{% endconfiguration %}
```

Comment:
This will handle this whole block + the YAML configuration isn't there anymore (in the parent PR).

```suggestion
{% include integrations/config_flow.md %}
```